int32 simulationType 
# 1 for Circular Spin
# 2 for Endurance Test
# 3 for Skid Pan
# 4 for Longitudinal Transition

